tests/build/_internal/test_cmake_file.py

```diff
@@ -7,60 +7,51 @@
 from tests.build._internal.config.factories import ConfigFactory
 from mbed_tools.build._internal.cmake_file import generate_mbed_config_cmake_file, _render_mbed_config_cmake_template
 
+@pytest.fixture()
+def fake_target():
+    target = dict()
```

Comment:
Why not just set the dictionary key-value pairs when it's constructed?

```
def fake_target():
    yield {"labels": ["foo"], ...}
```

Reply:
I'll try it and see if the line wrapping still looks ok.